test/lib/utils.js

```diff
@@ -111,7 +111,7 @@ function assertMetadata (t, obj) {
   t.ok(Array.isArray(_process.argv), 'process.title should be an array')
   t.ok(_process.argv.length >= 2, 'process.title should contain at least two elements')
   t.ok(/\/node$/.test(_process.argv[0]), `process.argv[0] should match /\\/node$/ (was: ${_process.argv[0]})`)
-  const regex = /(\/test\/(test|truncate|abort|edge-cases|lib\/unref-client)\.js|node_modules\/\.bin\/tape)$/
+  const regex = /(\/test\/.*\.js|node_modules\/\.bin\/tape)$/
```

Comment:
Reviewer note: I could revert this if you like. The reason for this is to not have arbitrary test breakage when running test files directly via `node test/foo.js`. Running test files like this isn't *typical*, but to help with test debugging I temporarily flirted with switching from `tape` to `tap` (https://node-tap.org/) which has better test-ending handling and isolation and allows running test files without the "runner" CLI.

If you look at the history on this file there are a number of changes coming in augmenting this regex to handle the next failure.

Reply:
👍 not too concerned about this check -- I'd be curious about the history of why it's here in the first place (but not SO curious that I'm going to track it down).